pkg/controller/storagecluster/cephobjectstores.go

```diff
@@ -83,6 +83,10 @@ func (r *ReconcileStorageCluster) createCephObjectStores(cephObjectStores []*cep
 // newCephObjectStoreInstances returns the cephObjectStore instances that should be created
 // on first run.
 func (r *ReconcileStorageCluster) newCephObjectStoreInstances(initData *ocsv1.StorageCluster, reqLogger logr.Logger) ([]*cephv1.CephObjectStore, error) {
+	gatewayInstances := int32(defaults.CephObjectStoreGatewayInstances)
+	if initData.Spec.ManagedResources.CephObjectStores.ManageGateway.GatewayInstances != 0 {
+		gatewayInstances = 1
```

Comment:
I think it should be:

```
gatewayInstances = initData.Spec.ManagedResources.CephObjectStores.ManageGateway.GatewayInstances
```

I think non-default value (>1) should be covered by an additional test case. Currently  it only covers `1`, so it wasn't caught.

Reply:
@egafford  -  I am misunderstanding this?

Reply:
this has been taken care of by the latest update